runtime/makelib/targets.mk.ftl

```diff
@@ -344,10 +344,10 @@ LIBCDEFS := $(wildcard /ztpf/$(UMA_ZTPF_ROOT)/base/lib/libCDEFSFORASM.so)
 
 # ddrgen preprocessor targets
 %.i : %.c
-	$(CC) $(CFLAGS) -E $< -o $@
+	$(CC) $(CFLAGS) -E $< | sed -n -e '/^@/p' > $@
```

Comment:
Please add a comment in here describing what the sed command is doing and why it improves the size.  This helps all future readers of the code.